Command failed: tsc -p ./tsconfig.json Unhandled promise rejection

## Type
What kind of issue is this?

<!-- please check the one that applies using an "x" -->
```
[x] Bug report.
[ ] Feature request.
```

## Current Behavior
The `react-undraw-cli` throw this:

> (node:5996) UnhandledPromiseRejectionWarning: Error: Command failed: tsc -p ./tsconfig.json
> (node:7172) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)

Catching that error and logging it, the correct error was:

> src/undraw/Undraw.test.tsx(2,22): error TS7016: Could not find a declaration file for module \'react-dom\'. \'\<myproject\>/node_modules/react-dom/index.js\' implicitly has an \'any\' type.\r\n  Try `npm install @types/react-dom` if it exists or add a new declaration (.d.ts) file containing `declare module \'react-dom\';`\r\nsrc/undraw/Undraw.test.tsx(6,1): error TS2582: Cannot find name \'it\'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.

## Expected behavior
At this point, it should generate the components, 

## Steps to Reproduce
- `mkdir react-undraw-issue`
- `cd react-undraw-issue`
- `npm i react-undraw@2.0.0-beta.1` (the error appears here too)
- add `"undraw": "react-undraw-cli",` in `scripts` of `package.json`
- `npm run undraw`

## Other Information
~After installing those `@types/...` in `_phantomChildren` field of `package.json` of your project, it creates the components as should.~
EDIT: idk any more what I did to make it work

@justinlettau I encoutered this problem in 2.0.0.

After I caught the error in scripts/build.js

```ts
try {
    await promiseExec('tsc -p ./tsconfig.json', { cwd: root });
  } catch (err) {
    console.error(err)
  }
```

this was the output

```
npm run build

> Investown@1.0.0 prebuild /Users/X/Development/Investown/investown-fe
> react-undraw-cli

Generating unDraw components ...
Component built: "UndrawAccessDenied"
Component built: "UndrawBankAccount"

Error: Command failed: tsc -p ./tsconfig.json

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:443:11)
    at Socket.emit (events.js:315:20)
    at Pipe.<anonymous> (net.js:674:12) {
  killed: false,
  code: 2,
  signal: null,
  cmd: 'tsc -p ./tsconfig.json',
  stdout: "../@types/react-native/globals.d.ts(40,15): error TS2300: Duplicate identifier 'FormData'.\n" +
    "../@types/react-native/globals.d.ts(85,5): error TS2717: Subsequent property declarations must have the same type.  Property 'body' must be of type 'string | ArrayBuffer | ArrayBufferView | Blob | FormData | URLSearchParams | ReadableStream<Uint8Array> | null | undefined', but here has type 'string | ArrayBuffer | DataView | Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | ... 8 more ... | undefined'.\n" +
    "../@types/react-native/globals.d.ts(94,5): error TS2717: Subsequent property declarations must have the same type.  Property 'signal' must be of type 'AbortSignal | null | undefined', but here has type 'AbortSignal | undefined'.\n" +
    "../@types/react-native/globals.d.ts(112,14): error TS2300: Duplicate identifier 'RequestInfo'.\n" +
    "../@types/react-native/globals.d.ts(131,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'Response' must be of type '{ new (body?: string | ArrayBuffer | ArrayBufferView | Blob | FormData | URLSearchParams | ReadableStream<Uint8Array> | null | undefined, init?: ResponseInit | undefined): Response; prototype: Response; error(): Response; redirect(url: string, status?: number | undefined): Response; }', but here has type '{ new (body?: string | ArrayBuffer | DataView | Int8Array | Uint8Array | Uint8ClampedArray | Int16Array | ... 8 more ... | undefined, init?: ResponseInit | undefined): Response; prototype: Response; error: () => Response; redirect: (url: string, status?: number | undefined) => Response; }'.\n" +
...
```

The list goes much longer.

As a workaround adding `"skipLibCheck": true` your the library's tsconfig compiler options seems to do the trick.

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -9,6 +9,7 @@
     "moduleResolution": "node",
     "outDir": "dist",
     "strict": true,
+    "skipLibCheck": true,
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"]
   },

